src/transformer.ts

```diff
@@ -133,6 +138,11 @@ export default class Transformer {
     let alternatives = lines.reduce<string[]>((result, inputType) => {
       if (inputType.type === 'String') {
         result.push(this.wrapWithZodValidators('z.string()', field, inputType));
+      } else if (Transformer.useDecimalLib && inputType.type === 'Decimal') {
+        result.push(
+          this.wrapWithZodValidators('z.instanceof(Decimal)', field, inputType),
+        );
+        this.addSchemaImport('Decimal');
```

Comment:
This might be a questionable decision, but I couldn't find any better way to add an `import { Decimal } from 'decimal.js'` for all schemas with Decimal fields